### PR TITLE
draft pull

### PR DESCRIPTION
**Pull request by smeea-2018**

### CODE REVIEW

**Comment by Nazehs** [comment]
Pls, move the scrips and CSS to the assets folder they should be in their respective folders not in the root folder.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (surajverma2587) [inline .vscode/settings.json:1]: Please remove this file by adding a `.gitignore` file in the root folder and add this code

```
.vscode
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+const generateBtn = document.querySelector("#generate");
```
Comment (surajverma2587) [inline index.js]: Would have been good to include this file in the `assets` folder within a `js` folder to keep the folder structure consistent.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  } // for ends here
+
+  return passwordGenerated.join("");
+};
+const generatePassword = () => {
+  // get the password length
+  const passwordLength = getPasswordLength();
+  if (passwordLength >= 8 && passwordLength <= 128) {
+    // get the password criteria
+    const passwordCriteria = getPasswordCriteria();
```
Comment (surajverma2587) [inline index.js]: Your code breaks here if the user answers no for all confirms. This needs to be fixed as you are losing marks here.

Comment (surajverma2587) [reply index.js]: You will need to check if the length of the `passwordCriteria` array is not `0` and only then proceed to generate a password else alert "Please select at least one criteria".

**Pull request merged**